test/apiv2/47-subnet-pools.at

```diff
@@ -0,0 +1,36 @@
+# -*- sh -*-
+#
+# subnet pools conversion tests
+#
+
+# Create a temporary containers.conf with custom DefaultSubnetPools
+tempdir=$(mktemp -d)
+cat > $tempdir/containers.conf <<EOF
+[network]
+default_subnet_pools = [
+  { "base" = "10.89.0.0/16", "size" = 24 },
+  { "base" = "10.90.0.0/15", "size" = 26 }
+]
+EOF
+
+# Start podman with custom config
+CONTAINERS_CONF=$tempdir/containers.conf
```

Comment:
This won't work, you must restart the service, i.e. something like that:
```
stop_service
CONTAINERS_CONF=$TESTS_DIR/containers.host-netns.conf start_service
```

And importantly once you are done you need to stop/start again so th enext tests no longe ruse that special config.

Reply:
ok let me fix that, thanks